app/views/course/statistics/assessments/_course_user.json.jbuilder

```diff
@@ -4,4 +4,5 @@ json.courseUser do
   json.name course_user.name
   json.role course_user.role
   json.isPhantom course_user.phantom?
+  json.email course_user.user.email
```

Comment:
This indirect relation is causing N+1 queries. Please fix.

Reply:
Fixed it by `Eager loading associations` as suggest by @syoopie

Reply:
Not fixed.

Reply:
Did I miss something ?
From the [reference](https://medium.com/doctolib/how-to-find-fix-and-prevent-n-1-queries-on-rails-6b30d9cfbbaf), I am understanding that the preload data will be re-used to avoid the N+1.

Reply:
Having new update for preloading user-email, and verify that the data for preload was re-used to avoid N+1.